test/cuda/test_execution_engine.cc

```diff
@@ -21,38 +21,44 @@
 #include <glog/logging.h>
 #include <gtest/gtest.h>
 
```

Comment:
we don't have `execution_engine` anymore, rename this test and/or merge somewhere?